testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/BrowserFlowTest.java

```diff
@@ -1238,6 +1253,259 @@ public void testLoginWithAlternativeOTPAndConditionalPassword(){
         }
     }
 
+    @Test
+    @AuthServerContainerExclude(REMOTE)
```

Comment:
My vote would be to move new stuff to separate class like "AllowDenyAuthenticatorConditionsTest" or something like this? It seems to me that BrowserFlowTest is already quite big and contains lots of stuff. WDYT?

Reply:
It makes sense. I agree with it.